example/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "notion-x-example",
-  "version": "4.16.2",
+  "version": "4.16.4",
```

Comment:
PRs to libraries generally don't update version numbers. they make changes and it's up to the package owner's discretion to update their respective version numbers